ast-nodes/variable-declaration.dart

```diff
@@ -79,7 +79,26 @@ class VariableDeclaration extends Declaration {
   }
 
   Pointer<LLVMOpaqueValue> generateCode(Module module) {
-    // TODO: implement
+    if (this.type is IntegerType) {
+      this.opaqueValue = llvm.LLVMBuildAlloca(
+          module.builder,
+          llvm.LLVMInt32TypeInContext(module.context),
```

Comment:
To avoid all these branches, use `VarType.getLlvmType()`. It is a method that's implemented on every (almost, for now) type and will return the right type. But don't forget to `.resolve()` the type – this way we automatically get support for named types